tests/AbstractIndexingTestCase.php

```diff
@@ -7,30 +7,45 @@ class AbstractIndexingTestCase extends TestCase
     /** @var Algolia_Algoliasearch_Helper_Algoliahelper */
     protected $algoliaHelper;
 
+    protected $indexPrefix;
+
+    /**
+     * @var array Default value of general store settings (not related to algolia's extention)
+     */
+    protected $defaultConfig = array(
+        'cataloginventory/options/show_out_of_stock' => '0',
+    );
+
+    const DEFAULT_PRODUCT_COUNT = 86;
```

Comment:
We could add one constant for each but the other ones are only used once. They are specific for the test, this one is what you get with the default config.